composer.json

```diff
@@ -4,7 +4,10 @@
   "require": {
     "php": ">=7.1",
     "spryker/kernel": "^3.0.0",
-    "spryker/locale": "^3.0.0"
+    "spryker/locale": "^3.0.0",
+    "spryker/propel-orm": "^1.6.0",
+    "propel/propel": "2.0.0-alpha8"
```

Comment:
*spryker/kernel* contains *spryker/propel-orm* and *propel/propel* as dependencies. Please remove them.